tests/test_cppgc.rs

```diff
@@ -49,62 +52,84 @@ fn cppgc_object_wrap() {
     }
   }
 
-  fn op_make_wrap(
+  fn op_wrap(
     scope: &mut v8::HandleScope,
-    _: v8::FunctionCallbackArguments,
+    args: v8::FunctionCallbackArguments,
     mut rv: v8::ReturnValue,
   ) {
-    let templ = v8::ObjectTemplate::new(scope);
-    templ.set_internal_field_count(2);
-
-    let obj = templ.new_instance(scope).unwrap();
-
-    let member = v8::cppgc::make_garbage_collected(
-      scope.get_cpp_heap().unwrap(),
-      Box::new(Wrap),
-    );
+    fn empty(
+      _scope: &mut v8::HandleScope,
+      _args: v8::FunctionCallbackArguments,
+      _rv: v8::ReturnValue,
+    ) {
+    }
```

Comment:
Maybe increment a counter here to ensure it's called the correct number of times?